### ISSUE TITLE
Stream `b` ended?

### ISSUE THREAD

**Issue by hemanth** [samccone/jsconf-2015-stream#1]
``` sh
○ → jsconf-2015-stream b
✨ Streaming track B✨ 
Possibly unhandled StatusCodeError: 404 - <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"/>
<title>404 - File or directory not found.</title>
<style type="text/css">
<!--
body{margin:0;font-size:.7em;font-family:Verdana, Arial, Helvetica, sans-serif;background:#EEEEEE;}
fieldset{padding:0 15px 10px 15px;} 
h1{font-size:2.4em;margin:0;color:#FFF;}
h2{font-size:1.7em;margin:0;color:#CC0000;} 
h3{font-size:1.2em;margin:10px 0 0 0;color:#000000;} 
#header{width:96%;margin:0 0 0 0;padding:6px 2% 6px 2%;font-family:"trebuchet MS", Verdana, sans-serif;color:#FFF;
background-color:#555555;}
#content{margin:0 0 0 2%;position:relative;}
.content-container{background:#FFF;width:96%;margin-top:8px;padding:10px;position:relative;}
-->
</style>
</head>
<body>
<div id="header"><h1>Server Error</h1></div>
<div id="content">
 <div class="content-container"><fieldset>
  <h2>404 - File or directory not found.</h2>
  <h3>The resource you are looking for might have been removed, had its name changed, or is temporarily unavailable.</h3>
 </fieldset></div>
</div>
</body>
</html>

    at new StatusCodeError (/usr/local/lib/node_modules/jsconf-2015-stream/node_modules/request-promise/lib/errors.js:26:15)
    at Request.RP$callback [as _callback] (/usr/local/lib/node_modules/jsconf-2015-stream/node_modules/request-promise/lib/rp.js:58:34)
    at Request.self.callback (/usr/local/lib/node_modules/jsconf-2015-stream/node_modules/request-promise/node_modules/request/request.js:368:22)
    at Request.emit (events.js:98:17)
    at Request.<anonymous> (/usr/local/lib/node_modules/jsconf-2015-stream/node_modules/request-promise/node_modules/request/request.js:1219:14)
    at Request.emit (events.js:117:20)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/jsconf-2015-stream/node_modules/request-promise/node_modules/request/request.js:1167:12)
    at IncomingMessage.emit (events.js:117:20)
    at _stream_readable.js:888:16
    at process._tickCallback (node.js:337:11)

```

We need to put a better message?

**Comment by samccone** [comment]
esh yeah that is rough, it seems like we _may_ be hitting their server too hard, any interest in opening a PR?

**Comment by hemanth** [comment]
yup, was playing with the code, was just planning to check the response code and then parse, any particular idea in your mind?

**Comment by samccone** [comment]
sounds good to me, as you can tell this is a super web scale project :cactus: :cake:

**Comment by hemanth** [comment]
heh heh :+1:  Will do a PR soon.

**Comment by hemanth** [comment]
Fixed at #2
